src/resources/views/configuration/users/partials/character.blade.php

```diff
@@ -1,4 +1,4 @@
-<a href="{{ route('character.view.default', ['character_id' => $character->character_id]) }}"
```

Comment:
please replace with `web::partials.character` if you're dropping tooltip, this blade is doing exactly the same.